examples/map/masking_hmi.py

```diff
@@ -56,34 +61,44 @@
 fig = plt.figure()
 ax = fig.add_subplot(projection=aia)
 aia.plot(axes=ax)
-aia.draw_contours(axes=ax, levels=200 * u.ct, colors="r")
+aia.draw_contours(axes=ax, levels=200, colors="r")
 for i in range(7):
     plt.text(*np.flip(regions[i].centroid), str(i), color="w", ha="center", va="center")
 
 ###############################################################################
-# Now let's plot those same regions on the reprojected HMI Map.
+# Now let's plot those same regions on the HMI Map.
 
 fig = plt.figure()
 ax = fig.add_subplot(projection=hmi)
 im = hmi.plot(axes=ax, cmap="hmimag", norm=Normalize(-1500, 1500))
-aia.draw_contours(axes=ax, levels=200 * u.ct, colors="r")
+aia.draw_contours(axes=ax, levels=200, colors="r")
 fig.colorbar(im)
 
 ###############################################################################
 # Now we have the regions, we need to create a new HMI map that masks out everything but the largest region.
 # To do so, we need to create the mask from the bounding box returned by `skimage`.
+# Note that we can do this from the thresholded region only because our AIA and HMI images are on the same pixel grid after reprojecting the AIA image.
 
 bbox = regions[0].bbox
 mask = np.ones_like(hmi.data, dtype=bool)
 mask[bbox[0]: bbox[2], bbox[1]: bbox[3]] = ~regions[0].image
 hmi_masked = sunpy.map.Map((hmi.data, hmi.meta), mask=mask)
 
 ###############################################################################
-# Finally, plot the largest HMI region.
+# We can then plot the largest HMI region.
 
 fig = plt.figure()
 ax = fig.add_subplot(projection=hmi_masked)
 im = hmi_masked.plot(axes=ax, cmap="hmimag", norm=Normalize(-1500, 1500))
 fig.colorbar(im)
 
+###############################################################################
+# Finally, we can plot the distribution of HMI LOS flux for only the unmasked values in the largest region shown above.
```

Comment:
I may be wrong, but isn't the  data is in units of magnetic field strength (Gauss) and not flux (Webers or similar)?

Reply:
Good point! I've changed it to "magnetic field".